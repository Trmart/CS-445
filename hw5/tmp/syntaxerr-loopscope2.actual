====================================
ERROR(10): Syntax error, unexpected "begin", expecting "!>" or "<=".
FILE: syntaxerr-loopscope2.c-
Number of errors: 1
Number of warnings: 0
