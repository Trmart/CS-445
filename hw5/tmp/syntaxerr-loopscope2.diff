2,5d1
< ERROR(10): Syntax error, unexpected "begin", expecting "<=".
< FILE: syntaxerr-loopscope2.c-
< Number of errors: 1
< Number of warnings: 0
