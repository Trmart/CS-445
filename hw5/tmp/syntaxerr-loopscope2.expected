====================================
ERROR(10): Syntax error, unexpected "begin", expecting "<=".
FILE: syntaxerr-loopscope2.c-
Number of errors: 1
Number of warnings: 0
